compiler/rustc_target/src/target_features.rs

```diff
@@ -226,6 +236,9 @@ const X86_ALLOWED_FEATURES: &[(&str, Stability)] = &[
     ("rdseed", Stable),
     ("rtm", Unstable(sym::rtm_target_feature)),
     ("sha", Stable),
+    ("sha512", Stable),
+    ("sm3", Stable),
+    ("sm4", Stable),
```

Comment:
oh, and same story as with the `amx_target_feature` set: these should not be stable. no recommendation on what to name this feature though.